Lab3.ipynb

```diff
@@ -0,0 +1,405 @@
+      "text/plain": [
+       "<PIL.JpegImagePlugin.JpegImageFile image mode=RGB size=302x75 at 0x7F40B5098AD0>"
+      ]
+     },
+     "execution_count": 115,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# 1.A.3:\n",
+    "Image.open('Lab3.1.jpg')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Problem 1.A.3: Using the equation above, we can take the integral from -infinity to -x and the integral from x to infintiy and set them equal to the probability of 5 sigma; since that's what we deem significant enough. This will yield what values of temperature are significant enough to be considered not created by the background. In order to find x we can utilize the equation (x-u)/(sigma). Therefore any temperature not in the range of 10-14 we can consider stastically significant enough to remove as bad data."
```

Comment:
Problem 1.A.3:
- minor typos: infintiy --> infinity
- though I see why you included the image of the full integration since the question asked to translate into math, it might be more effective to just describe the math just in terms of integration bounds and why we integrate instead of showing the full equation. In my explanation I talked about "threshold" for this